scripts/premigrate_created_modified.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import logging
+
+import django
+from django.db import connection, transaction
+
+from framework.celery_tasks import app as celery_app
+from scripts import utils as script_utils
+
+
+logger = logging.getLogger(__name__)
+
+ADD_COLUMNS = [
+    'ALTER TABLE osf_basefilenode ADD COLUMN created timestamp with time zone;',
+    'ALTER TABLE osf_basefilenode ADD COLUMN modified timestamp with time zone;',
+    "ALTER TABLE osf_blacklistguid ADD COLUMN created timestamp with time zone;",
+    "ALTER TABLE osf_blacklistguid ADD COLUMN modified timestamp with time zone;",
+    'ALTER TABLE osf_fileversion ADD COLUMN created timestamp with time zone;',
+    'ALTER TABLE osf_fileversion ADD COLUMN modified timestamp with time zone;',
+    "ALTER TABLE osf_guid ADD COLUMN modified timestamp with time zone;",
+    'ALTER TABLE osf_nodelog ADD COLUMN created timestamp with time zone;',
+    'ALTER TABLE osf_nodelog ADD COLUMN modified timestamp with time zone;',
+    "ALTER TABLE osf_pagecounter ADD COLUMN created timestamp with time zone;",
+    "ALTER TABLE osf_pagecounter ADD COLUMN modified timestamp with time zone;",
+]
+
+POPULATE_COLUMNS = [
```

Comment:
These are run by individual `run_sql` celery jobs. Couple notes:

* Setting `statement_timeout` as below kills queries after 10s, to avoid any locking issues.
* The `TO_TIMESTAMP(MAGIC)` converts BSON id's to timezoned timestamps. Jan 1 1970 is the default if this info is unavailable
* The subselect is necessary to `LIMIT 100` these queries, as `UPDATE` [does not support `LIMIT`](https://www.postgresql.org/docs/9.6/static/sql-update.html)